submissions/Ant-C-tech/dom-api-task/js/app.js

```diff
@@ -808,21 +77,30 @@ function hideContent() {
 }
 
 function changeContent(event) {
-    hideContent()
-    for (let item of mapAreas) {
-        item.style.fill = '#000'
-    }
-    event.target.style.fill = '#00b0ff'
-    const target = getTargetId(event)
-    contentSection.addEventListener('transitionend', function () {
-        contentSection.innerHTML = ''
-        showContent(content[target])
-    }, { once: true })
+    //Remove all clickListeners for animation time - solution for problem with fast change content
 
-}
-
-function getTargetId(event) {
-    return event.target.id
+    if (event.target.classList.contains('land-active')) {
+        return false
+    } else {
+        //Map visualization
+        for (const land of mapAreas) {
+            land.classList.remove('land-active')
+        }
+        map.removeEventListener('click', changeContent)
+        event.target.classList.toggle('land-active')
```

Comment:
`classList.add` i think will be more understandable in this case. Do the same, but you don't need any `toggling` here. Previously you removed all active classes, so main feature of toggle will not used